includes/wc-stock-functions.php

```diff
@@ -83,7 +83,8 @@ function wc_maybe_reduce_stock_levels( $order_id ) {
  * @param int $order_id
  */
 function wc_reduce_stock_levels( $order_id ) {
-	$order = wc_get_order( $order_id );
+	$order      = wc_get_order( $order_id );
+	$data_store = WC_Data_Store::load( 'order' );
```

Comment:
You can use [`$order->get_data_store()`](https://github.com/woocommerce/woocommerce/blob/master/includes/abstracts/abstract-wc-data.php#L157) to get the data store.

Also, there is now an extra unneccessary `set_stock_reduced` [here](https://github.com/woocommerce/woocommerce/blob/bd92f993591e4ef08d67b6bcc8245f4e50af7143/includes/wc-stock-functions.php#L75) that should be removed.